lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
@dchancogne good point.   Let's not overcomplicate, and yes that should be an API concern.